client/asset/dcr/dcr.go

```diff
@@ -759,6 +784,30 @@ func (dcr *ExchangeWallet) feeRate(confTarget uint64) (uint64, error) {
 	return 1 + uint64(atomsPerKB)/1000, nil // dcrPerKB * 1e8 / 1e3
 }
 
+// externalFeeEstimator gets the fee rate from the external API
+func externalFeeEstimator(nb uint64) (float64, error) {
+	url := externalApiUrl + "/utils/estimatefee?nbBlocks=" + strconv.Itoa(int(nb))
+	r, err := http.Get(url)
+	if err != nil {
+		return 0, err
+	}
+	body, err := ioutil.ReadAll(r.Body)
+	r.Body.Close()
+	if err != nil {
+		return 0, err
+	}
+	c := make(map[uint64]json.RawMessage)
```

Comment:
The responses look like `{"1":0.0001}`, so why not a `map[uint64]float64` and then

```go
estimatedFeeRate, ok := c[nb]
if !ok {
	return 0, errors.New("no fee rate for requested number of blocks")
```

https://go.dev/play/p/m3WtFtbKIKK

Does the value sometimes include a string or other non-float value?